src/views/product-downloads-view/boundary/components/install-callout/index.tsx

```diff
@@ -10,33 +10,36 @@ import MobileDownloadStandaloneLink from 'components/mobile-download-standalone-
 import Heading from 'components/heading'
 import { IconDownload16 } from '@hashicorp/flight-icons/svg-react/download-16'
 // Types
-import { DesktopClientProps, ReleaseBuild } from './types'
+import { InstallProps, ReleaseBuild } from './types'
 // Local imports
-import { operatingSystemIcons } from '../'
-import { getFileExtension, humanArch } from './helpers'
+import { operatingSystemIcons } from '..'
+import { getFileExtension, humanArch } from '../helpers'
 // Styles
-import s from './desktop-client-callout.module.css'
+import s from './install-callout.module.css'
 import { ContentWithPermalink } from 'views/open-api-docs-view/components/content-with-permalink'
 import viewStyles from 'views/product-downloads-view/product-downloads-view.module.css'
-import { BoundaryDesktopClient } from 'views/product-downloads-view/components/downloads-section/types'
+import { HTMLAttributes, ReactNode } from 'react'
+import { ReactElement } from 'react-markdown/lib/react-markdown'
 
 /**
  * Render a callout to download the Boundary Desktop Client.
  */
-function DesktopClientCallout({
-	desktopClientProps,
+function InstallCallout({
+	customInstallProps,
 	headingData,
+	className,
```

Comment:
maybe call this `cardClassName` to make it clear that the class name applies to the card?